tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -36,6 +37,27 @@
 )
 
 
+def document_download_response(override={}):
+    # See response from
+    # https://github.com/cds-snc/notification-document-download-api/blob/master/app/upload/views.py
+    base = {
```

Comment:
the [link above ](https://github.com/cds-snc/notification-document-download-api/blob/master/app/upload/views.py#L41) doesn't show a response of `mime_type`, `file_size` and `file_extension` unless I'm missing something

Reply:
Not yet because the PR https://github.com/cds-snc/notification-document-download-api/pull/35 hasn't been merged yet, but they will be there after